src/cobra/io/mat.py

```diff
@@ -479,42 +518,36 @@ def annotations_to_mat(
     """
     providers_used = set()
     for i in range(len(annotation_list)):
-        if annotation_list[i]:
-            providers_used.update(annotation_list[i].keys())
+        providers_used.update(annotation_list[i].keys())
     providers_used = providers_used.intersection(DICT_REPLACE[d_replace].keys())
     providers_used = list(providers_used)
     annotation_matlab = {prov: DICT_REPLACE[d_replace][prov] for prov in providers_used}
-    empty_lists = [[""] * len(annotation_list) for x in annotation_matlab]
+    empty_lists = [[""] * len(annotation_list) for _ in annotation_matlab]
     annotation_cells_to_be = dict(zip(annotation_matlab.values(), empty_lists))
     for i in range(len(annotation_list)):
-        if annotation_list[i]:
-            for provider_key, v in annotation_list[i].items():
-                if isinstance(v, str):
-                    v = [v]
-                if provider_key == "pubmed":
-                    v = ", ".join(
-                        [
-                            "PMID:" + annot if "PMID:" not in annot else annot
-                            for annot in v
-                        ]
-                    )
-                elif provider_key == "CHEBI":
-                    v = ", ".join(
-                        [
-                            "CHEBI:" + annot if "CHBEI:" not in annot else annot
-                            for annot in v
-                        ]
-                    )
-                elif provider_key == "ec-code":
-                    v = " or ".join(v)
-                else:
-                    v = ", ".join(v)
-                if provider_key not in providers_used:
-                    continue
-                annotation_cells_to_be[annotation_matlab[provider_key]][i] = v
+        for provider_key, v in annotation_list[i].items():
```

Comment:
Better name for `v`. It's not clear from the context what that is.

Reply:
Fixed.